ChangeSet
  1.1537 04/03/30 10:25:48 boutcher@brule.rchland.ibm.com +1 -0
  fix reference to pci_dma_mapping_error

  drivers/scsi/ibmvscsi/ibmvscsis.c
    1.5 04/03/30 10:25:11 boutcher@brule.rchland.ibm.com +1 -1
    fix reference to pci_dma_mapping_error

diff -Nru a/drivers/scsi/ibmvscsi/ibmvscsis.c b/drivers/scsi/ibmvscsi/ibmvscsis.c
--- a/drivers/scsi/ibmvscsi/ibmvscsis.c	Tue Mar 30 23:41:09 2004
+++ b/drivers/scsi/ibmvscsi/ibmvscsis.c	Tue Mar 30 23:41:09 2004
@@ -1863,7 +1863,7 @@
 					  queue->size * sizeof(*queue->msgs),
 					  PCI_DMA_BIDIRECTIONAL);
 
-	if (pci_dma_error(queue->msg_token))
+	if (pci_dma_mapping_error(queue->msg_token))
 		goto map_failed;
 
 	rc = plpar_hcall_norets(H_REG_CRQ, adapter->dma_dev->unit_address, queue->msg_token, PAGE_SIZE);
.........................................................................
# vim: syntax=diff

